honeybee_radiance_command/pcomb.py

```diff
@@ -0,0 +1,92 @@
+"""pcomb command."""
+
+from .options.pcomb import PcombOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Pcomb(Command):
+    """pcomb command."""
+
+    __slots__ = ('_input')
+
+    def __init__(self, options=None, output=None, input=[]):
+        """Command.
+
+        Args:
+            options: Command options. It will be set to Radiance default values if not
+                provided by user.
+            output: File path to the output file (Default: None).
+            input: A list of paths to radiance generated hdr images. (Default: []).
+        """
+        Command.__init__(self, output=output)
+        if options:
+            self._options = options
+        else:
+            self._options = PcombOptions()
+        self._input = input
+
+    @property
+    def options(self):
+        """pcomb options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = PcombOptions()
+
+        if not isinstance(value, PcombOptions):
+            raise ValueError('Expected PcombOptions not {}'.format(type(value)))
+
+        self._options = value
+
+    @property
+    def input(self):
+        """A list of paths to radiance generated hdr images."""
+        return self._input
+
+    @input.setter
+    def input(self, value):
+        if isinstance(value, list):
+            hdr_check = [image[-4:].lower() == '.hdr' for image in value]
+            hdrs = hdr_check.count(True)
+            if hdrs == len(value):
+                image_paths = [typing.normpath(path) for path in value]
+                joined_paths = ' '.join(image_paths)
+                self._input = joined_paths
+        elif not value:
+            self._input = []
+        else:
+            raise ValueError(
+                'A list of .hdr files required. Instead got %.' % (value)
+            )
```

Comment:
I think this can be simplified by doing the check first and then doing the rest.

```suggestion
        if not value:
            self._input = []
        elif not isinstance(value, (list, tuple)):
            value = [value]
        for image in value:
            if image[-4:].lower() != '.hdr':
                raise ValueError(
                    'A list of .hdr files required. Instead got %.' % (image)
                )            
        self._inputs = ' '.join(typing.normpath(path) for path in value)
```